client/trino-jdbc/src/test/java/io/trino/jdbc/TestJdbcWarnings.java

```diff
@@ -95,82 +96,72 @@ public void setupServer()
         }
     }
 
-    @AfterClass(alwaysRun = true)
+    @AfterAll
     public void tearDownServer()
             throws Exception
     {
         server.close();
         server = null;
```

Comment:
not need to null the server with Junit 5

Reply:
I'll leave all those for further cleanup later. We need to do a sweep through the codebase to update the BeforeAll/AfterAll calls anyway.